boxsdk/client/client.py

```diff
@@ -317,6 +318,40 @@ def create_group(self, name):
         )
 
     @api_call
+    def get_recent_items(self, limit=None, marker=None, fields=None):
```

Comment:
We should add `**kwargs`, and pass that to `MarkerBasedObjectCollection`. And then remove `return_full_pages=True` below, since that's a decision that the client, not the SDK, should make.

Depending on your personal preference, you could remove `limit`, `marker`, and `fields`, and leave them all for `**kwargs`. Or you can leave it as-is.